src/components/InvertedFlatList/index.js

```diff
@@ -52,13 +56,75 @@ class InvertedFlatList extends React.Component {
     componentWillUnmount() {
         this.list.getScrollableNode()
             .removeEventListener('wheel', this.invertedWheelEvent);
+        this.list.getScrollableNode()
+            .removeEventListener('keydown', this.invertedKeyboardScrollEvent);
```

Comment:
Kinda feels like we are throwing good money after bad here. IMO this feature is not worth it for the hack (and went unnoticed for months and months so obvious is not very important).

Basically, we are back to the `FlatList` inverted behavior being weird again.

I would suggest we:

1. Close this PR
2. Wait for this `react-native-web` issue here to be resolved

https://github.com/necolas/react-native-web/issues/2233

The code in this PR was copied from here anyway... https://github.com/staltz/manyverse/commit/4e0cfd62cec31a8992d3e247766bc6da1a4c6652

Reply:
@parasharrajat It seems we decide to fix this issue in upstream, should I close this PR?

Reply:
Yup sure.